routing/pathfind.go

```diff
@@ -415,6 +415,10 @@ type RestrictParams struct {
 	// OutgoingChannelID is the channel that needs to be taken to the first
 	// hop. If nil, any channel may be used.
 	OutgoingChannelID *uint64
+
+	// CltvLimit is the maximum time lock of the route excluding the final
+	// ctlv.
```

Comment:
Add a sentence explaining how the final cltv will affect the final route?

Reply:
Added